scripts/upload-package.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash -e
+
+#===================================================================================
+# Usage:
+#   uploadPackage <package_file> <ssh_command>
+#
+# Description:
+#   Uploads package file to the remote machine using SSH
+#   and runs specified command remotely.
+#
+# Globals:
+#   MANAGER_USER - (optional) Cloudify Manager SSH username (default: centos)
+#   MANAGER_IP   - (required) Cloudify Manager IP address
+#   SSH_KEY_PATH - (required) Cloudify Manager SSH key path
+
+# Arguments:
+#   $1 (package_file) - path to package file, eg. "./stage.tar.gz"
+#   $2 (ssh_command)  - string with command to be executed after uploading
+#                       on remote machine, eg. "tar xzf stage.tar.gz"
+# Returns:
+#   None
+#===================================================================================
+uploadPackage() {
+    MANAGER_USER=${MANAGER_USER:-centos}
+    PACKAGE_PATH=$1
+    SSH_COMMAND=$2
+    COMMON_OPTIONS="-o StrictHostKeyChecking=no"
+    ERROR_CODE=0
+
+    pwd
+
+    # Manager IP check
+    if [ -z $MANAGER_IP ]; then
```

Comment:
I will just verify it in one if and print out what the user entered

Reply:
Don't understand. Verify MANAGER_IP, SSH_KEY_PATH and PACKAGE_PATH in one if?

Reply:
yes, because you don't want to run the script multiple of times to find out that you had forgotten to fill in one of the three

Reply:
You don't have to run it multiple times. Exit is in the last if checking error code.

Reply:
yes, but also a single msg for all three will provide the same info for using this script